CMakeLists.txt

```diff
@@ -550,19 +550,16 @@ if (BUILD_PYTHON)
       if (SPHINX_EXECUTABLE)
         find_python_module (sphinx 1.1)
         find_python_module (numpydoc 0.9) # v0.7 & v0.8 have bugs for citations
-        if (USE_NBSPHINX)
-          find_program (PANDOC_EXECUTABLE NAMES pandoc DOC "a universal document converter")
-          if (PANDOC_EXECUTABLE)
-            find_python_module (nbsphinx)
-            find_python_module (IPython)
-          else ()
-            message (STATUS "Could NOT find Pandoc executable")
-          endif ()
-          if (NOT IPYTHON_FOUND OR SPHINX_VERSION_STRING VERSION_LESS 1.5.1)
-            set (NBSPHINX_FOUND FALSE CACHE BOOL "")
-            message (STATUS "Disabling notebook documentation examples (need pandoc/nbsphinx/ipython)")
-          endif ()
-        endif ()
+        # the sphinx-gallery extension
+        if (USE_SPHINX_GALLERY)
+            find_python_module (sphinx_gallery 0.7)
+        else ()
+            message (STATUS "Could NOT find shpinx-gallery extension")
+        endif()
+        if (NOT SPHINX_GALLERY_FOUND OR SPHINX_GALLERY_VERSION_STRING VERSION_LESS 0.7)
+            set (SPHINX_FOUND FALSE)
+            message (STATUS "Disabling documentation (need sphinx_gallery >= 0.7)")
+        endif()
```

Comment:
seems we also need all the optional dependencies the examples may use (hmat, pandas, libxml, ...) to enable the doc

Reply:
also SPHINX_GALLERY_FOUND implies version>=0.7 when calling find_python_module (sphinx_gallery 0.7)